deno_typescript/lib.rs

```diff
@@ -204,6 +204,8 @@ pub fn mksnapshot_bundle(
   js_check(
     isolate.execute(&bundle_filename.to_string_lossy(), bundle_source_code),
   );
+  // instantiate the bundle and delete __proto__ for security reasons
+  // this is intentionally not compliant with ECMA-262 Annex B.2.2.1.
```

Comment:
This comment should be undone?

Reply:
Bah. I thought I had done that. Will do.

Reply:
Resolved.